File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/system/CacheRevisionInfo.java
package nta.med.data.model.ihis.system;

public class CacheRevisionInfo {
	private String tableName;
	private Integer revision;
	public CacheRevisionInfo(String tableName, Integer revision) {
		super();
		this.tableName = tableName;
		this.revision = revision;
	}
	public String getTableName() {
		return tableName;
	}
	public void setTableName(String tableName) {
		this.tableName = tableName;
	}
	public Integer getRevision() {
		return revision;
	}
	public void setRevision(Integer revision) {
		this.revision = revision;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/System/CacheRevisionInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.System
{
    [Serializable]
    public class CacheRevisionInfo
    {
        private String _tableName;
        private Int32 _revision;

        public String TableName
        {
            get { return this._tableName; }
            set { this._tableName = value; }
        }

        public Int32 Revision
        {
            get { return this._revision; }
            set { this._revision = value; }
        }

        public CacheRevisionInfo() { }

        public CacheRevisionInfo(String tableName, Int32 revision)
        {
            this._tableName = tableName;
            this._revision = revision;
        }

    }
}